fuel-core/Cargo.toml

```diff
@@ -19,7 +19,11 @@ test = false
 
 [dependencies]
 anyhow = "1.0"
-async-graphql = { version = "=2.9", features = ["chrono", "chrono-tz", "tracing"] }
+async-graphql = { version = "=2.9", features = [
```

Comment:
Does `async-graphql` need the extra `=` in `version = "=2.9"` , it doesn't seem to be impacting anything but stylistically it may be nice to remove

Reply:
Probably does not matter, but remoted `=` as it seems unnecesary.